tests/performance/response_performance.py

```diff
@@ -0,0 +1,1304 @@
+<VALUE.NAMEDINSTANCE><INSTANCENAME CLASSNAME="TST_ResponseStressTestCxx"><KEYBINDING NAME="Id"><KEYVALUE VALUETYPE="string">83</KEYVALUE></KEYBINDING></INSTANCENAME><INSTANCE CLASSNAME="TST_ResponseStressTestCxx" ><PROPERTY NAME="Id"  TYPE="string"><VALUE>83</VALUE></PROPERTY><PROPERTY NAME="SequenceNumber"  TYPE="uint64"><VALUE>83</VALUE></PROPERTY><PROPERTY NAME="Pattern"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY><PROPERTY NAME="interval"  TYPE="uint64"><VALUE>36</VALUE></PROPERTY><PROPERTY NAME="S1"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY></INSTANCE></VALUE.NAMEDINSTANCE>
+<VALUE.NAMEDINSTANCE><INSTANCENAME CLASSNAME="TST_ResponseStressTestCxx"><KEYBINDING NAME="Id"><KEYVALUE VALUETYPE="string">84</KEYVALUE></KEYBINDING></INSTANCENAME><INSTANCE CLASSNAME="TST_ResponseStressTestCxx" ><PROPERTY NAME="Id"  TYPE="string"><VALUE>84</VALUE></PROPERTY><PROPERTY NAME="SequenceNumber"  TYPE="uint64"><VALUE>84</VALUE></PROPERTY><PROPERTY NAME="Pattern"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY><PROPERTY NAME="interval"  TYPE="uint64"><VALUE>22</VALUE></PROPERTY><PROPERTY NAME="S1"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY></INSTANCE></VALUE.NAMEDINSTANCE>
+<VALUE.NAMEDINSTANCE><INSTANCENAME CLASSNAME="TST_ResponseStressTestCxx"><KEYBINDING NAME="Id"><KEYVALUE VALUETYPE="string">85</KEYVALUE></KEYBINDING></INSTANCENAME><INSTANCE CLASSNAME="TST_ResponseStressTestCxx" ><PROPERTY NAME="Id"  TYPE="string"><VALUE>85</VALUE></PROPERTY><PROPERTY NAME="SequenceNumber"  TYPE="uint64"><VALUE>85</VALUE></PROPERTY><PROPERTY NAME="Pattern"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY><PROPERTY NAME="interval"  TYPE="uint64"><VALUE>34</VALUE></PROPERTY><PROPERTY NAME="S1"  TYPE="string"><VALUE>abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890abcdefghighjklmnopqrstuvwxyz01234567890</VALUE></PROPERTY></INSTANCE></VALUE.NAMEDINSTANCE>
+<VALUE.NAMEDINSTANCE><INSTANCENAME CLASSNAME="TST_ResponseStressTestCxx"><KEYBINDING NAME="Id"><KEYVALUE VALUETYPE="string">86</KEYVALUE></KEYBINDING></INSTANCENAME><INSTANCE CLASSNAME="TST_ResponseStressTestCxx" ><PROPERTY NAME="Id"  TYPE="string"><VALUE>86</VALUE></PROPERTY><PROPERTY NAME="SequenceNumber"  TYPE="uint64"><VALUE>86</VALUE></PROPERTY><PROPERTY NAME="Pattern"  TYPE="string"><VALUE>abcdefghig
```

Comment:
Seems the flake8 in py3.7 with minimum package levels does not recognize the underscore and its special meaning. Either remove the assignment, or find out which flake8 version has the support so we can upgrade the minimum version.

Reply:
Removed the  _